spec/models/notification_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe Notification, type: :model do
```

Comment:
In V1 there was no notification/activity table 🤔 
Just users, posts, comments and friendships. Lets start with it, please:)